DEVELOPMENT.md

```diff
@@ -0,0 +1,41 @@
+# Developing Stairway
+
+## Branching and Versioning
+The initial design of Stairway branches was that we would use two main branches in github.
+
+The **master** branch ws intended to be used for distributing code
```

Comment:
a slightly different approach in common-lib(it does not use master, just use develop):
`snapshot-local` is published after every PR merge
`libs-release-local` is triggered when manually make a github release. 

Just a note, not a blocker of this PR